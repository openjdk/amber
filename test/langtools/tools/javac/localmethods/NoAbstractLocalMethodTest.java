/*
 * @test /nodynamiccopyright/
 * @bug 8233038
 * @summary Tests need to be written for local methods feature
 * @compile/fail/ref=NoAbstractLocalMethodTest.out -XDrawDiagnostics NoAbstractLocalMethodTest.java
 */

public class NoAbstractLocalMethodTest {
    public static void main(String [] args) {
        abstract void xxx (int y);
    }
}
