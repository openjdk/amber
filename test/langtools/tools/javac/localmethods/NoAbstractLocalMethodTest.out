NoAbstractLocalMethodTest.java:10:18: compiler.err.expected3: class, interface, enum
1 error
